doc/developing.rst

```diff
@@ -12,50 +12,41 @@ Install Morepath for development
 
 .. highlight:: sh
 
-First make sure you have virtualenv_ installed for Python 3.5.
+Clone Morepath from github and go to the morepath directory::
 
-.. _virtualenv: https://pypi.python.org/pypi/virtualenv
+  $ git clone git@github.com:morepath/morepath.git
+  $ cd morepath
 
-Now create a new virtualenv somewhere for Morepath development::
+Make sure you have virtualenv_ installed for Python 3.5.
 
-  $ virtualenv /path/to/ve_morepath
+.. _virtualenv: https://pypi.python.org/pypi/virtualenv
 
-You should also be able to recycle an existing virtualenv, but this
-guarantees a clean one. Note that we skip activating the environment
-here, as this is just needed to initially bootstrap the Morepath
-buildout.
+Create a new virtualenv inside the morepath directory::
 
-Clone Morepath from github and go to the morepath directory::
+  $ virtualenv env
 
-  $ git clone git@github.com:morepath/morepath.git
-  $ cd morepath
+Activate the virtualenv::
 
-Now we need to run bootstrap.py to set up buildout, using the Python from the
-virtualenv we've created before::
+  $ source env/bin/activate
 
-  $ /path/to/ve_morepath/bin/python bootstrap.py
+Install the various dependencies and tools from requirements/develop.txt::
 
-This installs buildout, which can now set up the rest of the development
-environment::
+  $ pip install -r requirements/develop.txt
```

Comment:
I think it's better here to have

```console
$ pip install -r requirements/develop.txt --src src
```

which installs the editable packages in the ``src`` directory (see the [docs](https://pip.pypa.io/en/stable/reference/pip_wheel/#cmdoption--src)).

Suppose you want to have more than one environments as @faassen said he'd like to have, say py27 and py35.
If you omit the ``--src`` option you'll end up with two different checkouts of Reg, one in ``path/to/py27/src/reg`` and the other in ``path/to/py35/src/reg``.